src/Tabs/Tabs.js

```diff
@@ -206,8 +206,8 @@ class Tabs extends Component {
 
       return React.cloneElement(tab, {
         key: index,
+        index: index,
```

Comment:
in `handleTabTouchTap` we were previously using `tabIndex`, which I removed. Tabbing was broken in the first few commits if a `value` is not specified.

I played around with `getTabs()` and other available helper functions but couldn't figure out a way to identify the tab in `handleTabTouchTap` without slapping another prop on the tab.